src/Tools/Dumper.php

```diff
@@ -57,7 +56,7 @@ private function __construct()
      * @param mixed $var      The variable to dump.
      * @param int   $maxDepth The maximum nesting level for object properties.
      */
-    public static function dump($var, int $maxDepth = 2): string
+    public static function dump(mixed $var, int $maxDepth = 2): string
```

Comment:
Just since this is being modified, I'd like to get rid of this class entirely: it has very little to do with the DBAL and at the same time depends on the `Proxy` library.

Personally, I mostly rely on general tools like IDE, Xdebug, etc, so I would never use this class for troubleshooting. On the other hand, tools like Symfony or Laravel must have much more advanced tools for improving the developer experience.

Reply:
Yes, let's remove it. We could recommend Symfony VarDumper as replacement.